Resources/doc/reference/installation.rst

```diff
@@ -15,24 +15,39 @@ Installation
     //...
         "sonata-project/classification-bundle": "dev-master",
     //...
+    
+    },
+    "require-dev": {       
+        // ...
+        "sonata-project/easy-extends-bundle": "dev-master"
```

Comment:
What about `composer require --dev sonata-project/easy-extends-bundle` (same for the other bundles)? So we don't need the version constraint